checkout module assignment/Merve/styles.css

```diff
@@ -0,0 +1,159 @@
+* {
+    box-sizing: border-box;
+}
+
+:root {
+    --marine-blue: #02295a;
+    --purplish-blue: #473dff;
+    --pastel-blue: #adbeff;
+    --light-blue: #eff5ff;
+    --strawberry-red: #ed3548;
+    --cool-gray: #9699ab;
+    --light-gray: #d6d9e6;
+    --magnolia: #f0f6ff;
+    --alabaster: #fafbff;
+    --white: #ffffff;
+    --black:  #000000;
+}
+
+body {
+    font-family: 'Ubuntu', sans-serif;
+    background-color: var(--light-blue);
+    margin: 0;
+    padding: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
```

Comment:
Use `min-height` instead of height as your content may overlap on different screen sizes.